<launch>
  <!-- Object file Name without extension-->
  <arg name="object"         default="drill" />

  <!-- SPSegmenterNode subscriber/publisher args -->
  <arg name="pcl_in"         default="/camera/depth_registered/points" />
  <arg name="pcl_out"        default="points_out"/>
  <arg name="poses_out"      default="poses_out"/>

  <node pkg="sp_segmenter" type="SPSegmenterNode" name="SPSegmenterNode" args="-p">  
	  <!-- SPSegmenterNode Node arg pass -->
	  <param name="cur_name"   type="str" value="$(arg object)" />

    <!-- SPSegmenterNode subscriber/publisher args pass -->
	  <param name="POINTS_IN"  type="str" value="$(arg pcl_in)"/>
    <param name="POINTS_OUT" type="str" value="$(arg pcl_out)"/>
    <param name="POSES_OUT"  type="str" value="$(arg poses_out)"/>
  </node>

  <group ns="SPSegmenterNode">
    <include file="$(find sp_segmenter)/launch/parameters.launch" />
  </group>
</launch>
